experiment/update-hook/main.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Why do we need to rename this to update-hook? Please include a commit that moves the code to this directory before you start modifying it.

Reply:
The hmac tool will also need to delete a hook, so I modified the `add-hook` to also be able to do deletion, accordingly the package name needs to be changed.

Reply:
I have changed the first commit to be "refactor add-hook tool to update-hook"